### DISCUSSION TITLE
https://docsaid.org/blog/update-docusaurus-to-3-6-0/

### DISCUSSION THREAD

**Discussion by giscus** [DocsaidLab/website#9] [Announcements]
# https://docsaid.org/blog/update-docusaurus-to-3-6-0/

排除更新過程中遇到的問題

https://docsaid.org/blog/update-docusaurus-to-3-6-0/

**Comment by frankelinli** [comment]
一直担心，随着Docusaurus里面文章越来越多，以后构建时间会不会达到一个小时？

**Reply by zephyr-sh** [reply]
這次發布的 3.6 版在編譯速度上確實提速非常多，或許再過一陣子就能排除中文不友善的問題。（或許吧？）
